DOCUMENTATION.md

```diff
@@ -0,0 +1,211 @@
+# Solid Lints Documentation
+
+## Table of contents:
+
+1. [avoid_global_state](#avoid_global_state)
+1. [avoid_late_keyword](#avoid_late_keyword)
+1. [avoid_non_null_assertion](#avoid_non_null_assertion)
```

Comment:
Generate correct numbers here?

```suggestion
1. [avoid_global_state](#avoid_global_state)
2. [avoid_late_keyword](#avoid_late_keyword)
3. [avoid_non_null_assertion](#avoid_non_null_assertion)
```

Reply:
Yeah, but having these numbers be readable in raw markdown would be nice.

Reply:
done